modules/configuration/src/main/java/org/apache/ignite/internal/configuration/asm/InnerNodeAsmGenerator.java

```diff
@@ -175,8 +176,8 @@ class InnerNodeAsmGenerator extends AbstractAsmGenerator {
     /** {@link ConstructableTreeNode#construct(String, ConfigurationSource, boolean)} method name. */
     private static final String CONSTRUCT_MTD_NAME = "construct";
 
-    /** {@link Field} to {@link FieldDefinition} map. */
-    private final Map<Field, FieldDefinition> dieldToDieldDefinitionMap = new HashMap<>();
+    /** Mapping for each configuration schema node to it's {@link FieldDefinition}. */
+    private final Map<Field, FieldDefinition> fieldToFieldDefinitionMap = new HashMap<>();
```

Comment:
Still not clear, but I know that this class is not documented properly, so I can't ask for too much.
_"it's FieldDefinition"_ doesn't say what this definition means, and where it is. It's like saying _"maps field to its string"_ instead of _"... to its name"_.

By the way, **"its"**, not **"it's"**